### DISCUSSION TITLE
Drucken: Einträge im Druckdialog sollen erhalten bleiben

### DISCUSSION THREAD

**Discussion by fanie-mak** [e-netze/webgis-community#43] [Ideas]
Bei mehreren Ausdrücken wäre es vorteilhaft, wenn die Einträge im Druckdialog erhalten bleiben. 

![image](https://user-images.githubusercontent.com/79254257/222069882-23767050-741f-4750-972d-27aab17bca0e.png)

LG

**Comment by gis-eni** [comment]
Stimmt ist vorteilhaft aber funktioniert eigentlich genau so. Hab das sogar im KAGIS Atlas nachvollzogen.
Werte bleiben beim nach erneutem Aufruf des Druckdialogs erhalten. Selbst wenn das Werkzeug zwischenzeitlich gewechselt oder das Drucklayout gewechselt wird:

![image](https://user-images.githubusercontent.com/62870568/225659167-55feca96-0413-48d0-bfbb-3aec77e57318.png)

Schließt man den Atlas bzw. lädt man ihn neu, ist das Dialogfeld wieder leer.
